dd-sdk-android/src/test/kotlin/com/datadog/android/error/internal/DatadogExceptionHandlerTest.kt

```diff
@@ -90,7 +90,8 @@ internal class DatadogExceptionHandlerTest {
             mockNetworkInfoProvider,
             mockTimeProvider,
             mockUserInfoProvider,
-            mockLogWriter
+            mockLogWriter,
+            null
```

Comment:
We do not pass the mock context because the WorkManager lib is definitely not made for testing :/ 
I tried for a while to mock the context to please the WorkManager lib but it's too much work for one single test.